torch/distributed/_composable/fsdp/fully_shard.py

```diff
@@ -53,6 +54,8 @@ def fully_shard(
     managed_modules = _get_managed_modules(module)
     params, buffers = _get_managed_states(managed_modules)
     _move_states_to_device(params, buffers, device, mesh_info)
+    if params:
+        state._fsdp_param_group = FSDPParamGroup(params, module, mesh_info, device)
 
```

Comment:
non-blocking question for buffers: are we not sharding them because they usually consume small memories? even in finetuning, it's ``nn.parameters`` (requries_grad=False) instead of buffers?

Reply:
Yep!. Buffers themselves usually are small, and moreover, since they are not trainable, they never have optimizer state or gradients.